src/main/java/com/crud/tasks/trello/client/TrelloClient.java

```diff
@@ -20,21 +20,25 @@
     private String trelloAppKey;
     @Value("${trello.app.token}")
     private String trelloToken;
-
+    @Value("${trello.app.name}")
+    private String trelloName;
 
     @Autowired
     private RestTemplate restTemplate;
 
-    public List<TrelloBoardDto> getTrelloBoards() {
-
-        URI url = UriComponentsBuilder.fromHttpUrl(trelloApiEndpoint + "/members/MSajkiewicz/boards")
+    private URI urlBuild() {
+        URI url = UriComponentsBuilder.fromHttpUrl(trelloApiEndpoint + "/members/" + trelloName + "/boards")
                 .queryParam("key", trelloAppKey)
                 .queryParam("token", trelloToken)
                 .queryParam("fields", "name,id")
                 .build().encode().toUri();
+        return url;
+    }
+
+    public List<TrelloBoardDto> getTrelloBoards() {
 
-        TrelloBoardDto[] boardsResponse = restTemplate.getForObject(url, TrelloBoardDto[].class);
-        System.out.println(url);
+        TrelloBoardDto[] boardsResponse = restTemplate.getForObject(urlBuild(), TrelloBoardDto[].class);
+        System.out.println(urlBuild());
 
         if(boardsResponse != null) {
             return Arrays.asList(boardsResponse);
```

Comment:
Optional o ktorym mowa w zadaniu to cos takiego
`Arrays.asList(Optional.ofNullable(boardresponse).orElse(new TrelloBoardDto[0])`